index.js

```diff
@@ -27,27 +27,46 @@ module.exports = {
     var options = config.options;
     var project = options.project;
 
+    // provide compatibility with the string format
+    function directiveStringsToLists(headerConfig) {
+      Object.keys(headerConfig).forEach(function(key) {
+        var policy = headerConfig[key];
+        if ( typeof policy === "string" || policy instanceof String ) {
+          console.log('Warning: Content Security Policy');
```

Comment:
Can you use `this.ui.writeLine` instead?  That way the addon should observe the `--silent` flag..